.vscode/settings.json

```diff
@@ -1,6 +1,5 @@
 {
   "prettier.documentSelectors": ["src/**/*.{ts,tsx}", "*.{js,json}"],
   "vitest.include": ["src/**/*.spec.ts"],
-  "githubPullRequests.ignoredPullRequestBranches": ["dev"],
-  "editor.folding": false
```

Comment:
This is my favourite part of the PR! I've missed folding

Reply:
haha, yeah I didn't realise this was committed. True purists might argue `.vscode/` has no place in git, though considering we all use it sensible defaults are a good idea

Reply:
I think it's not a bad idea to have it committed and share settings for the project. 

I mentioned it as I was trying to fix this issue during the week as I was trying to fold code and didn't think to check here